static/app/views/alerts/rules/metric/ruleForm.tsx

```diff
@@ -123,6 +128,17 @@ class RuleFormContainer extends AsyncComponent<Props, State> {
     const {project} = this.state;
     // SearchBar gets its tags from Reflux.
     fetchOrganizationTags(this.api, organization.slug, [project.id]);
+
+    if (this.props.location.query.preset) {
+      const preset = PRESET_AGGREGATES.find(
+        p => p.id === this.props.location.query.preset
+      );
+      if (preset) {
+        preset
```

Comment:
@Neo-Zhixing do we have analytics when this happens?

Reply:
Added analytics for when people click on the "Use Template" button